webhook-app/webhooks.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This module contains functions that are called whenever a particular
+GitHub webhook is received."""
+
+import logging
+
+import github_helper
+import webhook_helper
+
+
+@webhook_helper.listen('ping')
+def pong(data):
+    return {'msg': 'pong'}
+
+
+def check_for_auto_merge_trigger(text):
+    """Checks the text for the phrases that should trigger an automerge."""
+    text = text.lower()
+
+    # The comment must address @dpebot directly.
+    if not '@{}'.format(github_helper.github_user()) in text:
+        return False
+
+    triggers = (
+        'merge on green',
+        'merge when travis passes',
+        'LGTM')
+
+    for trigger in triggers:
+        if trigger in text:
+            return True
+
+
+@webhook_helper.listen('issue_comment')
+def acknowledge_merge_on_travis(data):
+    """When a user comments on a pull request with one of the automerge
```

Comment:
Maybe add a link to docstring?

`See https://developer.github.com/v3/activity/events/types/`?